<?xml version="1.0"?>
<launch>

  <!-- load hera description -->
 <include file="$(find hera_description)/launch/load_description.launch">
    <arg name="robot_model" default="hera_full"/>
  </include>

  <!-- DYNAMIXEL CONTROLLER -->
  <include file="$(find hera_head)/launch/dynamixel_controller_head.launch"/>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <rosparam param="source_list" subst_value="true">["head/joint_states"]</rosparam>
  </node>

 <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

  <!-- <node pkg="hera_head" type="joint_state.py" name="head_joint_state"/> -->

  <!-- HEAD INTERFACE -->
  <node name ="head_interface" pkg="hera_head" type="head.py" output="screen"/>

</launch>
